frontend/src/component/project/Project/ProjectStatus/ProjectLifecycleSummary.tsx

```diff
@@ -0,0 +1,177 @@
+                            }}
+                        />
+                    </Counter>
+                    <span>flags in pre-live</span>
+                </p>
+                <Stats>
+                    <dt>Avg. time in stage</dt>
+                    <dd>18 days</dd>
+                </Stats>
+            </LifecycleBox>
+            <LifecycleBox>
+                <p>
+                    <Counter>
+                        <BigNumber>2</BigNumber>
+
+                        <FeatureLifecycleStageIcon
+                            aria-hidden='true'
+                            stage={{
+                                name: 'live',
+                                enteredStageAt: '',
+                                environments: [],
+                            }}
+                        />
+                    </Counter>
+                    <span>flags in live</span>
+                </p>
+                <Stats>
+                    <dt>Avg. time in stage</dt>
+                    <dd>10 days</dd>
+                </Stats>
+            </LifecycleBox>
+            <LifecycleBox>
+                <p>
+                    <Counter>
+                        <BigNumber>6</BigNumber>
+
+                        <FeatureLifecycleStageIcon
+                            aria-hidden='true'
+                            stage={{
+                                name: 'completed',
+                                enteredStageAt: '',
+                                environments: [],
+                                status: 'kept',
+                            }}
+                        />
+                    </Counter>
+                    <span>
+                        <LinkNoUnderline
+                            to={`/projects/${projectId}/placeholder`}
+                        >
+                            flags
+                        </LinkNoUnderline>{' '}
+                        in cleanup
+                    </span>
+                </p>
+                <Stats>
+                    <dt>Avg. time in stage</dt>
+                    <dd>
+                        <NoData>No data</NoData>
+                    </dd>
+                </Stats>
+            </LifecycleBox>
+            <LifecycleBox>
+                <p>
+                    <Counter>
+                        <BigNumber>15</BigNumber>
+
+                        <FeatureLifecycleStageIcon
+                            aria-hidden='true'
+                            stage={{
+                                name: 'archived',
+                                enteredStageAt: '',
+                            }}
+                        />
+                    </Counter>
+                    <span>flags in archived</span>
+                </p>
+                <Stats>
+                    <dt>This month</dt>
+                    <dd>3 flags archived</dd>
+                </Stats>
+            </LifecycleBox>
+        </Wrapper>
+    );
```

Comment:
This is the complete, rolled-out version of what it'll look like. We'll very likely refactor this in a future iteration..